### DISCUSSION TITLE
What do you usually delete to free up some storage?

### DISCUSSION THREAD

**Discussion by itislu** [itislu/42free#10] [Feedback & Ideas]
# These are the default arguments at the moment:

> [!TIP]
> **Post the whole path in the comments so it can be added as a default argument!**

## Linux
| Default Arguments - Linux | Program | Program Type |
|-------------------|-------------------|-------------------|
| $HOME/.cache | _Cache_ | System |
| $HOME/.local/share/docker | Docker | Virtualization |
| $HOME/.config/Code/Cache | VSCode | Code Editor |
| $HOME/.config/Code/CachedData | VSCode | Code Editor |
| $HOME/.config/Code/CachedExtensionVSIXs | VSCode | Code Editor |
| $HOME/.config/Code/Crashpad | VSCode | Code Editor |
| $HOME/.config/Code/User/workspaceStorage | VSCode | Code Editor |
| $HOME/.config/Code/Service Worker | VSCode | Code Editor |
| $HOME/.config/Code/WebStorage | VSCode | Code Editor |
| $HOME/.config/Cursor/Cache | Cursor | Code Editor |
| $HOME/.config/Cursor/CachedData | Cursor | Code Editor |
| $HOME/.config/Cursor/CachedExtensionVSIXs | Cursor | Code Editor |
| $HOME/.config/Cursor/Crashpad | Cursor | Code Editor |
| $HOME/.config/Cursor/User/workspaceStorage | Cursor | Code Editor |
| $HOME/.config/Cursor/Service Worker | Cursor | Code Editor |
| $HOME/.config/Cursor/WebStorage | Cursor | Code Editor |
| $HOME/.var/app/com.discordapp.Discord | Discord | Communication App |
| $HOME/.var/app/com.slack.Slack | Slack | Communication App |
| $HOME/.var/app/com.brave.Browser/cache | Brave | Browser |
| $HOME/.var/app/com.brave.Browser/config/BraveSoftware/Brave-Browser/Default/Service Worker | Brave | Browser |
| $HOME/.var/app/com.google.Chrome/cache | Chrome | Browser |
| $HOME/.var/app/com.google.Chrome/config/google-chrome/Default/Service Worker | Chrome | Browser |
| $HOME/.config/google-chrome/Default/Service Worker | Chrome | Browser |
| $HOME/.var/app/com.opera.Opera/cache | Opera | Browser |
| $HOME/.var/app/com.opera.Opera/config/opera/Default/Service Worker | Opera | Browser |
| $HOME/.var/app/com.vivaldi.Vivaldi/cache | Vivaldi | Browser |
| $HOME/.var/app/com.vivaldi.Vivaldi/config/vivaldi/Default/Service Worker | Vivaldi | Browser |
| $HOME/.brew | Homebrew | Package Manager |
| $HOME/.capt | capt | Package Manager |

---

## macOS
| Default Arguments - macOS | Program | Program Type |
|-------------------|-------------------|-------------------|
| $HOME/Library/Caches | _Cache_ | System |
| $HOME/Library/Containers/com.docker.docker/Data/vms/0/data | Docker | Virtualization |
| $HOME/Library/Application Support/Code/Cache | VSCode | Code Editor |
| $HOME/Library/Application Support/Code/CachedData | VSCode | Code Editor |
| $HOME/Library/Application Support/Code/CachedExtensionVSIXs | VSCode | Code Editor |
| $HOME/Library/Application Support/Code/Crashpad | VSCode | Code Editor |
| $HOME/Library/Application Support/Code/User/workspaceStorage | VSCode | Code Editor |
| $HOME/Library/Application Support/Code/Service Worker | VSCode | Code Editor |
| $HOME/Library/Application Support/Code/WebStorage | VSCode | Code Editor |
| $HOME/Library/Application Support/Cursor/Cache | Cursor | Code Editor |
| $HOME/Library/Application Support/Cursor/CachedData | Cursor | Code Editor |
| $HOME/Library/Application Support/Cursor/CachedExtensionVSIXs | Cursor | Code Editor |
| $HOME/Library/Application Support/Cursor/Crashpad | Cursor | Code Editor |
| $HOME/Library/Application Support/Cursor/User/workspaceStorage | Cursor | Code Editor |
| $HOME/Library/Application Support/Cursor/Service Worker | Cursor | Code Editor |
| $HOME/Library/Application Support/Cursor/WebStorage | Cursor | Code Editor |
| $HOME/Library/Application Support/discord/Cache | Discord | Communication App |
| $HOME/Library/Application Support/discord/Service Worker | Discord | Communication App |
| $HOME/Library/Application Support/Slack/Cache | Slack | Communication App |
| $HOME/Library/Application Support/Slack/Service Worker | Slack | Communication App |
| $HOME/Library/Application Support/BraveSoftware/Brave-Browser/Default/Service Worker | Brave | Browser |
| $HOME/Library/Application Support/Google/Chrome/Default/Service Worker | Chrome | Browser |
| $HOME/Library/Application Support/Opera Software/Opera Stable/Service Worker | Opera | Browser |
| $HOME/Library/Application Support/Vivaldi/Default/Service Worker | Vivaldi | Browser |

**Comment by alexandregv** [comment]
Hi, as shared on Slack, here are the values used by 42Paris workstations in a script similar to 42free:
```
$HOME/.cache
$HOME/.config/Code/Cache
$HOME/.config/Code/CachedData
$HOME/.config/Code/CachedExtensionVSIXs
$HOME/.config/Code/Service Worker
$HOME/.config/Slack/Cache
$HOME/.config/Slack/Service Worker
$HOME/.config/google-chrome/Default/Service Worker
$HOME/.config/BraveSoftware/Brave-Browser/Default/Service Worker
$HOME/.config/discord/Cache
$HOME/.config/discord/Code Cache
$HOME/.config/discord/Service Worker
$HOME/.config/.vagrant.d/boxes
```

**Reply by itislu** [reply]
Thanks a lot!
